src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,23 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  categories: [],
+  status: 'Under construction',
+};
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState,
+  reducers: {
+    checkStatus(state) {
+      return {
+        ...state,
+        status: 'Under Construction',
+      };
+    },
+  },
```

Comment:
- [ ] Please checkStatus reducer should accept an action  so the user can control the under construction state

Reply:
Fixed.